### PR TITLE
Loginauthentication

### PR DESCRIPTION
**Pull request by phanit1**
Completed all tasks

### CODE REVIEW

File: create.py [hunk 1]
```diff
@@ -43,4 +45,40 @@ def register():
...
+        user=session['email']
+        return render_template("login.html")
+    except:
+        var1 = "You must log in to view the homepage"
+        return render_template("reg.html",var1 = var1)
+
+@app.route('/logout')
+def logout():
+    try:
+        user=session['email']
```
Comment (ashishk144) [inline create.py]: why are you storing the email on logout? any specific reason?

File: templates/login.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<html>
+    <head>
+        <link rel = "stylesheet" href="{{ url_for('static',filename='reg.css') }}">
+    </head>
+    <body>
+        <h1 style="text-align:center">Login Success</h1>
+        <form>
+            <button type="submit" class="registerbtn" formaction = "http://127.0.0.1:5000/logout"><b>Logout</b></button>
```
Comment (ashishk144) [inline templates/login.html]: you can skip the http://127.0.0.1:5000 and enter only the URI for the form action.

**Comment by ashishk144** [comment]
Looks like application.py and reg.py files are unused, drop them off from the folder.

**Comment by phanit1** [comment]
If i delete the two files, then i can merge bro

**Review by ashishk144 — COMMENTED** [review]
> If i delete the two files, then i can merge bro

You can after resolving the other comments too.

File: create.py (hunk 1 above)
Comment (phanit1) [reply create.py]: sorry it s written by mistake i have been taken off that line

File: templates/login.html (hunk 2 above)
Comment (phanit1) [reply templates/login.html]: did it bro

**Pull request merged**